### ISSUE TITLE
Solved! Desktop client can not connect after password has been changed

### ISSUE THREAD

**Issue by wehkah** [nextcloud/desktop#366]
I am using NextCloud 13.0.1 and several devices synchronize data with it. On my Linux PC I am runninx Linux Mint and I used to use the OwnCloud desktop client, but I just learned that now there is a NextCloud desktop client as well, so I installed this one. But the following error is true for both of the desktop clients:

Everything worked fine until I used the web interface and changed my user’s password. After the change, the Linux desktop client can’t connect any more, although it asks me for the new password. Every other device, no matter if Android or pc, can access the server as before using the new password.

In the servers log file, I can see that the client wants to connect, so there is no firewall or router issue involved. Even more strange: if I fire up a virtual machine with a live Linux called Knoppix, the OwnCloud desktop client connects without any error to the server. 

What is wrong there? Why does the desktop client on my PC have this problem, while no other connecting client does not have it?

**Comment by wehkah** [comment]
Ok, I think I found the solution: right before I changed the password, I also upgraded to NC 13.0.2. In this upgrade process, the brute force app had been deactivated, without any notice. After the upgrade, me pc somehow was black listed, but since the app was deactivated, there was no way of seeing that on the security tab in admin mode. After I reactivated the app, I was able to white list my pc and now it can synch like before. 

So, there seems to be a bug in the upgrade process!
